src/Aspire.Dashboard/Components/Controls/GridValue.razor.cs

```diff
@@ -37,7 +37,7 @@ public partial class GridValue
     public EventCallback<bool> IsMaskedChanged { get; set; }
 
     [Parameter]
-    public int? MaxDisplayLength { get; set; }
+    public int? MaxDisplayLength { get; set; } = 8;
```

Comment:
I don't think we want to set that default to 8, because the view itself decides whether to truncate by checking to see if MaxDisplayLength has a value:

```razor
<span class="cellText" title="@(ToolTip ?? Value)">
    @(MaxDisplayLength.HasValue ? TrimLength(Value) : Value)
</span>
```